homeassistant/components/lektrico/entity.py

```diff
@@ -0,0 +1,30 @@
+"""Entity classes for the Lektrico integration."""
+from __future__ import annotations
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import LektricoDeviceDataUpdateCoordinator
+from .const import DOMAIN
+
+
+class LektricoEntity(CoordinatorEntity[LektricoDeviceDataUpdateCoordinator]):
+    """Define an Lektrico entity."""
+
+    _attr_has_entity_name = True
+
+    def __init__(
+        self,
+        coordinator: LektricoDeviceDataUpdateCoordinator,
+        friendly_name: str,
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self._attr_device_info = DeviceInfo(
+            identifiers={(DOMAIN, str(coordinator.serial_number))},
```

Comment:
Same question, what type is the serial number?

Reply:
I changed it. Thanks.